Grabber.pro

```diff
@@ -33,8 +33,7 @@ unix:!macx{
 	desktop.files += release/Grabber.desktop
 
 	icon.path = $$PREFIX/share/pixmaps
-	icon.extra = cp resources/icon.png resources/Grabber.png
-	icon.files += resources/Grabber.png
+	icon.extra = install -D -m644 resources/icon.png $(INSTALL_ROOT)/$$PREFIX/share/pixmaps/Grabber.png

```

Comment:
How do you mean an absolute path? if `INSTALL_ROOT` isn't set it should be empty, if `PREFIX` isn't set it should be set to `/usr/local` automatically which would give us the string `//usr/local` which should reduce to `/usr/local` atleast on my system.